### ISSUE TITLE
Readme.md

### ISSUE THREAD

**Issue by wiredone** [Hylozoic/hylo-evo#852]
Where it says, "So long as this repo remains private, remaining docs are available in Confluence."..  Please release the docs too!

**Comment by tibetsprague** [comment]
Good point! Most of those docs are out of date though and need lots of work….
